angular-video-crime-miner/src/app/app-routing.module.ts

```diff
@@ -4,7 +4,7 @@ import { ViewComponent } from './view/view.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
 import { MediaComponent } from './media/media.component';
 import { ReportsComponent } from './reports/reports.component';
-import { UploadComponent } from './upload/upload.component';
```

Comment:
I changed this to lowercase so it would build on Docker in a Linux container. The uppercase/lowercase stuff breaks it on linux due to different operating system stuff

Reply:
Look in the "files changed" tab for more details on this on my review, for some reason it doesn't show both the old and new  here, only the old!